frontend/sass/_global.scss

```diff
@@ -26,4 +26,9 @@ ul {
 
 h1, h2, h3 {
   font-weight: 700;
-}
\ No newline at end of file
+}
+
+a {
+  color: inherit !important;
+  text-decoration: none !important;
```

Comment:
`!important` is really necessary here? If so, it would be nice to add a simple comment explaining why

Reply:
Yeah, somehow the `a` tag don't obey the global setting without he !important :/